packages/removable-tags/src/components/styledComponent/StyledTag.js

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+import { colors as colorConstants } from "@crave/farmblocks-theme";
+
+const StyledTag = styled.div`
+  display: inline-block;
+  box-sizing: border-box;
+  border: 1px solid ${colorConstants.GREY_16};
+  border-radius: 4px;
+  padding: 6px 0px 8px 8px;
+  background-color: ${colorConstants.INDIGO_MILK_CAP};
+  height: 32px;
+  margin: 2px;
+
+  font-size: 14px;
+  color: white;
+  font-family: Lato;
+  font-weight: 600;
+
+  cursor: default;
+  &:hover {
+    border: 2px solid ${colorConstants.GREY_32};
+    padding: 5px 0px 7px 7px;
+    background-color: ${colorConstants.BLUE_CORN};
+  }
```

Comment:
The interaction is a bit odd here. The tag gets a hover effect when the cursor is anywhere over the tag. But the only interactive part is the X:
![2018-03-09 16 52 04](https://user-images.githubusercontent.com/1459283/37228300-e9064036-23be-11e8-9fca-0d623809cfa3.gif)

The hover effect should happen only when the cursor is over the X:
![2018-03-09 16 53 15](https://user-images.githubusercontent.com/1459283/37228345-0bad3450-23bf-11e8-833f-f038af37d23d.gif)

You can achieve this by adding `pointer-events: none;` to the container and adding `pointer-events: auto;`  to the icon.